Libraries/Alert/Alert.js

```diff
@@ -28,18 +28,11 @@ type Options = {
   onDismiss?: ?Function,
 };
 
-type AlertType = $Keys<{
-  default: string,
-  'plain-text': string,
-  'secure-text': string,
-  'login-password': string,
-}>;
+/* 'default' | plain-text' | 'secure-text' | 'login-password' */
+type AlertType = string;
 
-export type AlertButtonStyle = $Keys<{
-  default: string,
-  cancel: string,
-  destructive: string,
-}>;
+/* 'default' | 'cancel' | 'destructive' */
+export type AlertButtonStyle = string;
```

Comment:
Good job!
But it would be better to keep this inside the spec file, because there is where we're defining types. We're could still exporting it here too or updating all call-sites that uses these types from here to the spec path.

What do you think about it?

Reply:
I agree with keeping types inside the spec file. I'll move types to spec file :+1: